### PR TITLE
update to clarify-go v0.3.0 (breaking)

### PR DESCRIPTION
**Pull request by smyrman**
List of changes:
- Update to use claify-go v0.3.0, which includes breaking changes.
- Use the new automationcli packge to define the automaton CLI.
- Add new example routine for the evaluate method.
- Add new example routine for sending message to slack.
- Update README instructions.
- Update default CI tasks.

### CODE REVIEW

**Comment by smyrman** [comment]
ping @sarzhann, just checking that you have seen this.

File: routines.go [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+			// Detect names of format "device-id/measurement-name/eng-unit".
+			// Note that production code may want to use regular expressions
+			// to specify more precise rules.
+			comps := strings.Split(item.Name, "/")
+			if len(comps) == 3 {
+				item.Name = comps[1]
+				item.EngUnit = comps[2]
+				item.Labels.Add("device-id", comps[0])
+			}
+
```
Comment (sarzhann) [inline routines.go]:
```suggestion
```

**Review by sarzhann — APPROVED** [review]
One small nit, otherwise LGTM 👍

File: routines.go (hunk 1 above)
Comment (smyrman) [reply routines.go]: Added & run gofmpt

**Pull request merged**